src/components/UnreadActionIndicator.js

```diff
@@ -1,29 +1,17 @@
-import React from 'react';
-import {Animated, View} from 'react-native';
-import PropTypes from 'prop-types';
+import React, {memo} from 'react';
+import {View} from 'react-native';
 import styles from '../styles/styles';
 import Text from './Text';
 
-const propTypes = {
-    // Animated opacity
-    // eslint-disable-next-line react/forbid-prop-types
-    animatedOpacity: PropTypes.object.isRequired,
-};
-
-const UnreadActionIndicator = props => (
-    <Animated.View style={[
-        styles.unreadIndicatorContainer,
-        {opacity: props.animatedOpacity},
-    ]}
-    >
+const UnreadActionIndicator = () => (
+    <View style={styles.unreadIndicatorContainer}>
         <View style={styles.unreadIndicatorLine} />
         <Text style={styles.unreadIndicatorText}>
             NEW
         </Text>
-    </Animated.View>
+    </View>
 );
 
-UnreadActionIndicator.propTypes = propTypes;
 UnreadActionIndicator.displayName = 'UnreadActionIndicator';
 
-export default UnreadActionIndicator;
+export default memo(UnreadActionIndicator);
```

Comment:
There's not a lot behind it. I read a bit about it and it seemed like a good idea, since we don't really. want to rerender the marker that often, right?

Reply:
@Gonals for further reading, [this article](https://dmitripavlutin.com/use-react-memo-wisely/) really walks through the use of `memo` and when to use it.

In general, I think this is a good guideline:

> Use the following rule of thumb: don’t use memoization if you can’t quantify the performance gains.

Reply:
Cool! Thanks guys! I'll remove it